draft-ietf-quic-qpack.md

```diff
@@ -84,9 +84,10 @@ code and issues list for this draft can be found at
 
 The QUIC transport protocol {{QUIC-TRANSPORT}} is designed to support HTTP
 semantics, and its design subsumes many of the features of HTTP/2 {{?RFC7540}}.
-HTTP/2 uses HPACK {{!RFC7541}} for header compression.  If HPACK were used for
-HTTP/3 {{HTTP3}}, it would induce head-of-line blocking due to built-in
-assumptions of a total ordering across frames on all streams.
+HTTP/2 uses HPACK {{!RFC7541}} for compression of the header and trailer
+sections.  If HPACK were used for HTTP/3 {{HTTP3}}, it would induce head-of-line
+blocking due to built-in assumptions of a total ordering across frames on all
```

Comment:
```suggestion
blocking for headers due to built-in assumptions of a total ordering across frames on all
```

Reply:
Or maybe better would be ", it would make headers head-of-line blocking due to..."